scratchpad/tn_api/tn.py

```diff
@@ -2,9 +2,10 @@
 import math
 from dataclasses import dataclass
 from typing import TypeVar, Generic, Iterable
+from qtree import np_framework
 
 class Array(np.ndarray):
-    shape: tuple[int]
```

Comment:
removed all instances of `tuple[...` because was getting error that types can't have `[`